spec/controllers/users/registrations_controller_spec.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+RSpec.describe Users::RegistrationsController do
+  let(:user) { create(:user) }
+  let(:password_new) { FFaker::String.from_regexp(User::PASSWORD_FORMAT_REGEX) }
+
+  before do
+    request.env['devise.mapping'] = Devise.mappings[:user]
+    sign_in user
+  end
+
+  describe 'GET #index' do
```

Comment:
It is not a `#index` action
```suggestion
  describe '#edit' do
```